src/lib/calendar/calendar-menu/calendar-menu.ts

```diff
@@ -32,6 +32,11 @@ declare global {
   }
 }
 
+/**
+ * The web component class behind the `<forge-calendar-menu>` custom element.
+ * 
+ * @tag forge-calendar-menu
+ */
```

Comment:
Is there any way (or reason why) we may want to mark a component as "internal"? This component is used within the calendar to help with separation of concerns, but isn't likely to be documented or used externally unless as part of another composite component. Just curious what your thoughts are on these ones?

Reply:
Looks like `@internal` and `@ignore` tags are [supported](https://custom-elements-manifest.open-wc.org/analyzer/getting-started/#supported-jsdoc) so I could definitely see if that excludes it from the manifest entirely or at least includes an `"internal": true` annotation.  The analyzer also does take an `--exclude` glob as an option if necessary.

I know Compodoc (which is used under the hood by the Angular Storybook integration) uses `@ignore` to exclude something from documentation, and TypeScript uses `@internal` to exclude things from the `.d.ts` declarations file if the `--stripInternal` option is provided.  If those two are identical to the analyzer, seems like `@internal` makes the most sense.

Reply:
I confirmed that `@internal` excludes the component from the manifest.